core/templates/dev/head/domain/dashboard/DashboardBackendApiService.js

```diff
@@ -33,7 +33,26 @@ oppia.factory('DashboardBackendApiService', [
     });
   };
 
+  var _fetchExplorationStats = function(explorationId) {
+    var requestUrl = (
+      '/dashboardhandler/explorationstats/?exp_id=' + explorationId);
+    return $q(function(resolve, reject) {
+      $http.get(requestUrl, {
+        cache: true
+      }).then(function(response) {
+        if (resolve) {
```

Comment:
@AllanYangZhou Please see latest code. I've implemented the simplest implementation - return the `$http.get()` promise directly from the service, and then use it inside controller as required!